{"id":13968,"className":"java.lang.IllegalArgumentException","methodName":"IllegalArgumentException","signature":"\u003cjava.lang.IllegalArgumentException: void IllegalArgumentException(java.lang.String,java.lang.Throwable)\u003e","javadoc":"/** \n * Constructs a new exception with the specified detail message and cause. \u003cp\u003eNote that the detail message associated with \u003ccode\u003ecause\u003c/code\u003e is \u003ci\u003enot\u003c/i\u003e automatically incorporated in this exception\u0027s detail message.\n * @param message the detail message (which is saved for later retrievalby the  {@link Throwable#getMessage()} method).\n * @param cause the cause (which is saved for later retrieval by the{@link Throwable#getCause()} method).  (A \u003ctt\u003enull\u003c/tt\u003e valueis permitted, and indicates that the cause is nonexistent or unknown.)\n * @since 1.5\n */\n","tf":{"freq":{"argument":1,"except":3,"detail":4,"save":2,"nonexist":1,"later":2,"messag":6,"incorpor":1,"permit":1,"link":2,"valuei":1,"unknown":1,"illeg":1,"automat":1,"specifi":1,"caus":6,"indic":1,"associ":1,"retriev":1,"method":2,"throwabl":2,"construct":1,"retrievalbi":1,"note":1},"maxFreq":6,"totalTerms":44},"loc":3}